src/SignhostAPIClient/Rest/SignhostApiReceiver.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Signhost.APIClient.Rest;
+using Signhost.APIClient.Rest.DataObjects;
+
+namespace Signhost.APIClient
+{
+	public class SignhostApiReceiver
+		: ISignhostApiReceiver
+	{
+		private readonly SignhostApiReceiverSettings settings;
+
+		public SignhostApiReceiver(SignhostApiReceiverSettings receiverSettings)
+		{
+			this.settings = receiverSettings;
+		}
+
+		public bool IsPostbackChecksumValid(
```

Comment:
Ensure `/// <inheritdoc />` documentation is set so the UI will display the documentation from the interface in this instance.